modules/data.atmosphere/NAMESPACE

```diff
@@ -109,6 +109,7 @@ export(temporal_downscale_half_hour)
 export(upscale_met)
 export(wide2long)
 export(write_noaa_gefs_netcdf)
+import(arrow)
```

Comment:
```# Generated by roxygen2: do not edit by hand```

You need to add arrow to the DESCRIPTION not the NAMESPACE

Reply:
removed arrow from namespace by removing imports call from roxygen